build/KoreBuild.ps1

```diff
@@ -91,13 +93,9 @@ if (!(Test-Path "$koreBuildFolder\Sake"))
     Invoke-WebRequest "https://dist.nuget.org/win-x86-commandline/v3.5.0-beta2/NuGet.exe" -OutFile "$koreBuildFolder/nuget.exe"
 }
 
-$makeFilePath = "makefile.shade"
-if (!(Test-Path $makeFilePath))
-{
-    $makeFilePath = "$koreBuildFolder\shade\makefile.shade"
-}
-
-Write-Host "Using makefile: $makeFilePath"
-
 $env:KOREBUILD_FOLDER=$koreBuildFolder
-&"$koreBuildFolder\Sake\0.2.2\tools\Sake.exe" -I $koreBuildFolder\shade -f $makeFilePath @args
+$makeFileProj = "$koreBuildFolder/targets/makefile.proj"
+if ($allparams) {
+    $targets = '/t:' + $allparams.Replace(' ', ';')
+}
+dotnet msbuild $makeFileProj /p:KoreBuildDirectory="$koreBuildFolder\" /p:RepositoryRoot="$repoFolder\" $targets
```

Comment:
I actually tried multiple targets and it works fine - primarily because the `$targets` is a variable and not code in the file with semicolons.